routes.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const app = express();
+const port = 8080;
+
+const data = [
+  {
+    id: 1,
+    item: "Toilet Paper",
+    person: "Kesha",
+    country: "USA",
+    image_url:
+      "https://images.news18.com/ibnlive/uploads/2023/03/whatsapp-image-2023-03-04-at-3.12.13-pm.jpeg?impolicy=website&width=510&height=356",
+    description:
+      "Kesha revealed on a talk show that she had a strange habit of eating toilet paper.",
+  },
+];
+
+app.use(express.json());
+
+app.get("/", (req, res) => {
```

Comment:
Consider adding error handling for your routes. This can help to catch and handle any errors that may occur during the execution of your route handlers. You can do this by wrapping your route handlers in a try-catch block or by using a middleware for error handling. [important]